lib/openstax/page.rb

```diff
@@ -0,0 +1,66 @@
+module Openstax
+  class Page
+    MATCH_PARAGRAPH = '//p[@id]'
+    MATCH_FIGURE = '//figure'
+    MATCH_SELECTOR_PARAGRAPH = 'p'
+    MATCH_SELECTOR_FIGURE = 'figure'
+    MATCH_FIGURE_VISIBLE_TEXT = './/figcaption'
```

Comment:
if a figure has text content inside of it, is it always inside a paragraph that would be indexed separately? wondering if there is any content we need to index for figures other than the caption

Reply:
@philschatz do you know the answer to this? ^^